### PR TITLE
feat(cxl-ui): center instructor image

### PR DESCRIPTION
**Pull request by anoblet**
https://app.clickup.com/t/86b00z797

cxl com_institute_dashboard_

### CODE REVIEW

File: packages/cxl-ui/scss/cxl-featured-course-card.scss [hunk 1]
```diff
@@ -132,7 +132,7 @@
       @media #{mq.$large} {
         position: absolute;
         bottom: calc(-1 * var(--lumo-space-xl));
-        right: calc(2 * var(--lumo-space-xl));
+        right: calc(27.5% - 200px);
```
Comment (lkraav) [inline packages/cxl-ui/scss/cxl-featured-course-card.scss:135]: This is still a lot of magic numbers :thinking: is there no way to have some canonical generic numberless positioning formula here?

Comment (pawelkmpt) [reply packages/cxl-ui/scss/cxl-featured-course-card.scss:135]: I like `left: calc(50% + 100px)` more. Easier to understand.

Looks good on `@media #{mq.$x-large}`
but doesn't look perfect `@media #{mq.$large}` (1024px) - too close to the right edge.

Screenshot 2024-04-17 at 18 50 51
Screenshot 2024-04-17 at 18 51 00

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

**Comment by pawelkmpt** [comment]
@anoblet ping

**Comment by lkraav** [comment]
> Although it may seem like more of a magic number, `right: calc(27.5% - 200px);` displays better across all sizes. The rational behind it was the middle ground between 1/4, and 1/3 minus half of the image width.

What about the option of JS inline style calculations based on actual image size?

**Review by pawelkmpt — APPROVED** [review]

**Comment by lkraav** [comment]
> It'd be perfect but this is suppose to be a quick task. We don't want to spend on it more than we already did. Proposed solution is going live.

What exactly was spent here? Majority of the time it's been sitting waiting for review. This «giving up» style is a sure way to build into tech debt.

**Pull request merged**